module-2/backend/Liam_barstad/week_four.txt

```diff
@@ -0,0 +1,54 @@
+* What is a cookie?
+
+  - A cookie is a piece of information stored in the user's browser that is not encrypted, for use
+    in controllers and other website mechanics.
+
+* What’s the difference between a session and a cookie?
+
+  - A session is encrypted to prevent tampering, a cookie is not.
+
+* What’s a flash and when do you want to use flashes?
+
+  - A flash is a preset key, similar to a parameter, that is used to contain specific messages that can
+    be displayed when certain actions are performed.
+
+* Why do people say “HTTP is stateless”?
+
+  - Every time a controller action is called, a new instance of that controller (and view(s)) is invoked.
+
```

Comment:
not quite. Because a stateless protocol does not require the server to retain session information or status about each communications partner for the duration of multiple requests. HTTP is a stateless protocol, which means that the connection between the browser and the server is lost once the transaction ends